x-pack/plugins/upgrade_assistant/public/application/components/kibana_deprecations/deprecation_item.tsx

```diff
@@ -0,0 +1,145 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React, { FunctionComponent } from 'react';
+import {
+  EuiAccordion,
+  EuiButton,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiButtonEmpty,
+  EuiText,
+  EuiCallOut,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+import type { DomainDeprecationDetails } from 'kibana/public';
+import { DeprecationHealth } from '../shared';
+import { LEVEL_MAP } from '../constants';
+import { StepsModalContent } from './steps_modal';
+
+const i18nTexts = {
+  getDeprecationTitle: (domainId: string) => {
+    return i18n.translate('xpack.upgradeAssistant.deprecationGroupItemTitle', {
+      defaultMessage: "'{domainId}' is using a deprecated feature",
+      values: {
+        domainId,
+      },
+    });
+  },
+  docLinkText: i18n.translate('xpack.upgradeAssistant.deprecationGroupItem.docLinkText', {
+    defaultMessage: 'View documentation',
+  }),
+  manualFixButtonLabel: i18n.translate(
+    'xpack.upgradeAssistant.deprecationGroupItem.fixButtonLabel',
+    {
+      defaultMessage: 'Show steps to fix',
```

Comment:
"Show steps to fix" and "Quick resolve" are an awkward pairing. 

Instead of a generic "Quick resolve" label, is it possible to provide an indication of what's going to happen/what you need to do? For example, "Migrate worksheets". If there's a way to do it automatically, the next prompt could list what's going to happen and include the Quick resolve button as CJ suggested. 

Unless it's really clear what's going to happen, I think people will be reluctant to choose "Quick resolve".

Reply:
That's a fair point. 

I opened https://github.com/elastic/kibana/issues/98482, which will allow the consumer to define an explanation of what the "automatic fix" will do. However, I don't think this would be applicable to the button label, and I'm also not sure it makes sense to add another field for the user to define something like this.

Do you have any other suggestions? Once https://github.com/elastic/kibana/issues/96060 is implemented, we will know whether the deprecation is config- or feature-related. I'm not sure if that would help here or not.

Also, in https://github.com/elastic/kibana/pull/97159#pullrequestreview-646231940, CJ suggested co-locating the manual steps and "quick resolve" button in the same flyout. In we went in this direction, I think there would only be a single initial entry point. /cc @dborodyansky